code/src/UI/Views/MainView.xaml

```diff
@@ -69,10 +69,9 @@
             <Grid Visibility="{Binding LoadingContentVisibility}">
                 <TextBlock Style="{StaticResource TextMainLoadingStyle}" Text="{x:Static res:StringRes.LoadingTemplatesText}" />
             </Grid>
-            <Grid Visibility="{Binding NoContentVisibility}">
-                <TextBlock Style="{StaticResource TextMainLoadingStyle}" Text="{x:Static res:StringRes.NoContentText}" />
+            <Grid Visibility="{Binding LoadedContentVisibility}">
```

Comment:
Could you not just do away with this outer grid and use the visibility binding on the inner textblock to control its display? 
Nested single elements with different visibility bindings are hard to debug and maintain.